src/components/create-vm-dialog/createVmDialogUtils.js

```diff
@@ -76,6 +82,14 @@ export function filterReleaseEolDates(os) {
     );
 }
 
+export function getOSDescription(os) {
+    if (os.eolDate && compareDates(ACCEPT_EOL_DATES_AFTER, os.eolDate) < 0)
+        return <span><ExclamationTriangleIcon /> {cockpit.format(_("Vendor support ended $0"), os.eolDate)}</span>;
```

Comment:
Did you keep the ACCEPT_EOL_DATES_AFTER on purpose here? That's the code path where libosinfo knows the EOL date, so it looks like this would accept "EOL < 1 year ago" as "new". That seems misleading to me?

Reply:
Yeah, good point. Let's change that to "EOL date is in the past".

Reply:
Yes, that's what I did.